src/grammar.grammar

```diff
@@ -105,13 +105,14 @@ lineEnd { newline | eof }
 }
 //!tokens
 @tokens {
-  @precedence {"mandatory", "optional", "alternative", "or", "constraints", "cardinality ","sum", "avg", "Integer", "Real","String", "Boolean","len", "-", "false", "true",real, number, identifier}
+  @precedence {"mandatory", "optional", "alternative", "or", "constraints", "cardinality ","sum", "avg", "Integer", "Real","String", "Boolean","len", "-", "false", "true",real, number, identifier, extIdentifier, id}
```

Comment:
- fix comma ("true",real)
- What is the reason for this precedence order?

Reply:
Thats a ranking if there are conflicts

Reply:
Yes, that is the meaning of precedence but HOW can there be such conflicts? These words may no be used as feature or property names, always appear in a set manner. So this does not make sense to me.